packages/manager/apps/dedicated/client/app/networkSecurity/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,34 @@
+{
+  "network_security_title": "Sécurité réseau",
+  "network_security_dashboard_title": "Tableau de bord Anti-DDoS (bêta)",
+  "network_security_dashboard_guides": "Guides",
+  "network_security_dashboard_header_content": "En surveillant constamment le trafic réseau, nous observons un certain nombre d’activités suspectes. Pour chacun d’entre eux, le trafic réseau entrant est redirigé de manière transparente via nos centres de nettoyage pour une analyse plus approfondie et un éventuel nettoyage des attaquants afin de fournir la meilleure expérience sans interruption.\nVeuillez noter que les adresses IP sources ne sont pas exposées ici car elles ne sont pas fiables pour la plupart des attaques (c'est-à-dire les attaques par usurpation d'identité ou par réflexion).",
```

Comment:
Instead of adding `\n` in the translation, this can be handled in the MTML by splitting between 2 elements.